src/restore_snapshot.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e 
+
+# Quit script if snapshot file doesn't exist
+
+if [ ! -f /snapshot.json ]; then
+    echo "runpod-worker-comfy: No snapshot file found. Exiting..."
+    exit 0
+fi
+
+cd /comfyui/
+
+# Install ComfyUI-Manager
+git clone https://github.com/ltdrdata/ComfyUI-Manager.git custom_nodes/ComfyUI-Manager
+
+cd custom_nodes/ComfyUI-Manager
+
+pip install -r requirements.txt
+
+mkdir startup-scripts
+mv /snapshot.json startup-scripts/restore-snapshot.json
+
+cd ../..
+
+# Trigger restoring of the snapshot by performing a quick test run
+# Note: We need to use `yes` as some custom nodes may try to install dependencies with pip
+/usr/bin/yes | python main.py --cpu --quick-test-for-ci
```

Comment:
Does this need to be python3?

Reply:
I think it's a good idea to add a symlink for python to point to python3.10. 

Here is the relevant section of the Dockerfile with the symlink added:
```
RUN apt-get update && apt-get install -y \
 python3.10 \
 python3-pip \
 git \
 wget \
 && ln -sf /usr/bin/python3.10 /usr/bin/python
```